kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -1,6 +1,8 @@
-ChannelRouter = Backbone.Router.extend({
+TopicChannelRouter = Backbone.Router.extend({
     initialize: function(options) {
+        _.bindAll(this);
         this.default_channel = options.default_channel;
+        $("#nav_learn").click(this.intercept_learn_nav);
```

Comment:
Hrm, now you mention it, we do already have a catch all splat that checks for navigation events. I should probably just intercept there instead.